src/NuGet/Microsoft.CodeAnalysis.CSharp.nuspec

```diff
@@ -4,6 +4,8 @@
     <id>Microsoft.CodeAnalysis.CSharp</id>
     <description>
       .NET Compiler Platform ("Roslyn") support for C#, Microsoft.CodeAnalysis.CSharp.dll.
+
+      More details at https://github.com/dotnet/roslyn/wiki/NuGet-packages
```

Comment:
Good idea. Thanks
Do we have a list or a known set of owners for aka.ms managed by Roslyn team?